examples/connect/off-host-custom-images/README.md

```diff
@@ -0,0 +1,24 @@
+## Off-Host Execution with Custom Images
+
+Uses `my-org/` as the docker image repository and `my-regcred` as
+the [registry credential / imagePullSecret](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)
+for this
+imaginary environment.
+
+> NOTE: the `launcher.templateValues.pod.imagePullSecrets` provides the pod-level `imagePullSecret` that provides
+> the secret to pull the `launcher.defaultInitContainer` as well.
+
+### Usage
+
+```bash
+helm template rsc rstudio/rstudio-connect -f values.yaml -f runtime-values.yaml
+```
+
+### `values.yaml`
+
+Most values for configuration, as
+discussed [here](https://docs.rstudio.com/helm/rstudio-connect/kubernetes-howto/appendices/arch_overview.html#remote-execution)
+
+### `runtime-values.yaml`
+
+Values for configuring the `runtime.yaml` images for session execution.
```

Comment:
We should do a better job standardizing on our terminology. We sometimes refer to these as "session", "content", and "runtime" images. This is probably worth having a broader discussion; we have similar inconsistencies with "off host" and "remote content" execution, and other concepts too, I'm sure.

Reply:
Well said. Maybe "`runtime.yaml` images for executing content"? 😛